server/src/main/resources/message/messages_en.properties

```diff
@@ -0,0 +1,28 @@
+CobolLineReaderImpl.longLineMsg=Source text cannot go past column 80
```

Comment:
This key pattern will work for now, but I would prefer storing by purpose, e.g. error/notification, not by class. We may review and change it later.